molecule/default/vars/podman.yml

```diff
@@ -0,0 +1,27 @@
+tas_single_node_podman:
+  policy: "{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}/../../vm-testing/containers/policy.json"
```

Comment:
In general, a better approach is having the variable contain the contents of the file as opposed to path to the file. This works better with AAP execution environments where users would more likely specify the value rather than handle creating the file.

If users want to source the value from a file, they can use https://docs.ansible.com/ansible/latest/collections/ansible/builtin/file_lookup.html

Reply:
I changed tas_single_node_podman.policy to simply contains policy.json content and extended meta/argument_specs.yml schema